module02/task01/src/middleware/auth.ts

```diff
@@ -0,0 +1,19 @@
+import { NextFunction, Request, Response } from "express";
+import Jwt from "jsonwebtoken";
+
+export function checkToken(request: Request, response: Response, next: NextFunction) {
+  console.log(request.path);
+  if (request.path === "/login") {
+    return next();
+  }
+  const token = request.headers["authorization"] as string;
+  if (!token) {
+    return response.status(401).send("Unauthorized");
+  }
+  try {
+    Jwt.verify(token, "secret");
```

Comment:
Please put secret in environment variables and reuse it

Reply:
agreed and made the changes